templates/service/build.sbt

```diff
@@ -0,0 +1,28 @@
+import TestPhases.oneForkedJvmPerTest
+import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+
+name := "$!APP_NAME!$"
+
+lazy val root = (project in file("."))
+  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
+  .configs(IntegrationTest)
+  .settings(inConfig(IntegrationTest)(Defaults.itSettings): _*)
+
+scalaSettings
+publishingSettings
+defaultSettings()
+AppDependencies.appDependencies
```

Comment:
I found this line confusing, we are calling appDependencies method that creates a sequence that modifies settingKeys? why not `libraryDependencies ++= libraryDependencies ++= compile ++ test` ?

Reply:
the reasoning was that `AppDependencies.appDependencies` can set up whatever dependency-related settings it wants - in some projects I've set up exclusions in there which involves settings other than `libraryDependencies`.

I don't mind changing it though if you like?

Reply:
@frankoid I've added a more complete example in the comments that also includes setting `libraryDependencies`. I could easily add exclusions there if I wanted. Not sure if there is sth else that wouldn't be possible but maybe optimising for that rare scenario is not needed?